src/frontend/test_runner/tests/testdata/subquery_expr_correlated.yaml

```diff
@@ -5,7 +5,7 @@
   logical_plan: |
     LogicalProject { exprs: [$1, $2], expr_alias: [x, y] }
       LogicalFilter { predicate: ($1 > $3) }
-        LogicalApply { type: LeftOuter }
+        LogicalApply { type: LeftOuter, on: always }
```

Comment:
Recommend using `TRUE` instead of `always`. May fix it later.

Reply:
This behavior is aligned with `LogicalJoin`. Maybe we can discuss it later.